src/models/observer.ts

```diff
@@ -0,0 +1,11 @@
+export interface IObserver {
+  update(): void;
+}
+
+export interface ISubject {
+  addListener(observer: IObserver): void;
```

Comment:
I'm using the interface `IObserver` along with functions like `addListener`. It sounds natural to me, but I wanted to double-check if this naming convention (using Observable and Listener in the same context) is fine.

Reply:
Yep this is ok, this might just go to some utils or shared stuff - it is not an Orb model, more like utility interface.